### PR TITLE
[CPORM-4] - Migrate to NPM registry

### PR DESCRIPTION
**Pull request by juancancela**
**Description**

Migration from Bower to NPM registry. [INFO](https://docs.google.com/document/d/1fRG1BH2nJDWfFoJBEYHFs6hUTvS_4Zq7jIDj_9U-IcE/edit#)

Notes:

- Removed bower.json file
- Added missing fields to package.json (main, authors, etc) and removed bower dependencies
- Updated a test that is making the build fail very often -unrelated to this change-
- Tested locally running Karma tests (requires installing locally karma **npm install -g karma-cli** and **then running karma start karma.conf.js --single-run**)

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,13 @@
 {
-  "name": "n-service-worker",
+  "version": "0.0.0",
+  "name": "@financial-times/n-service-worker",
```
Comment (debugwand) [inline package.json:3]: 👍

File: package.json [hunk 2]
```diff
@@ -27,6 +31,10 @@
       "env"
     ]
   },
+  "authors": [
```
Comment (debugwand) [inline package.json:34]: both these people have actually moved or left.

**Comment by debugwand** [comment]
on the karma needing globally installed, do you think we can get it installed by putting karma and karma-cli in devDependencies?

File: package.json (hunk 2 above)
Comment (juancancela) [reply package.json:34]: I moved them from bower file to keep history, but if it makes more sense I can completely remove the authors key from file.

Comment (debugwand) [reply package.json:34]: maybe leave in rhys?

**Comment by debugwand** [comment]
one other thing - the README might need updated, and if you could add a line about what the command is to install it, that would be an improvement on what is there currently

File: test/integration/cache.spec.js [hunk 3]
```diff
@@ -401,7 +401,7 @@ describe('cache', () => {
 					.then(cache => cache.set(testUrl))
 					.then(() => new DB('requests', { dbName: 'next:test-cache'}).get(testUrl))
 					.then(inDb => {
-						expect(inDb.expires).to.be.closeTo(now + (60 * 1000), 1000);
+						expect(inDb.expires).to.be.closeTo(now + (60 * 1000), 2000);
```
Comment (rashedul-hassan-app) [inline test/integration/cache.spec.js:404]: Is it making it more reliable?

**Review by rashedul-hassan-app — APPROVED** [review]
Looks good to me. +1 on the README as Dawn suggested.

**Optional**: Also, looking at it, since now you don't make use of the global Karma, might also be a good idea to add this to the docs so users are aware that they need to have `node_modules` populated before running the tests.

File: test/integration/cache.spec.js (hunk 3 above)
Comment (juancancela) [reply test/integration/cache.spec.js:404]: yes, based on some local tests I ran, many results were like 1054, 1008, 1040 ms. 2000 should be fine.

**Pull request merged**